tests/control/test_fetch_mapping.py

```diff
@@ -0,0 +1,84 @@
+import json
+from pathlib import Path
+from unittest import mock
+
+from rest_framework.exceptions import NotAuthenticated, PermissionDenied
+
+from control.api import fetch_mapping
+from pytest import raises
+
+FIXTURES_DIR = Path(__file__).resolve().parent / "fixtures"
+
+fhirConceptMap = {
+    "id": "cm_gender",
+    "group": [
+        {
+            "element": [
+                {"code": "F", "target": [{"code": "female", "equivalence": "equal"}]},
+                {"code": "M", "target": [{"code": "male", "equivalence": "equal"}]},
+            ]
+        }
+    ],
+}
+
+
+class MockResponse:
```

Comment:
does anyone know if there's a more "pythonic" way to mock this ?

Reply:
ref [https://docs.pytest.org/en/stable/monkeypatch.html#monkeypatching-returned-objects-building-mock-classes](https://docs.pytest.org/en/stable/monkeypatch.html#monkeypatching-returned-objects-building-mock-classes)
`MockResponse` looks ok.
The patch below could be done in a fixture (cf the last example of the paragraph)